recipes/torchquad/LICENSE.txt

```diff
@@ -0,0 +1,1347 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
The package from GitHub already includes the license, so you shouldn't include it manually in the recipe.

Reply:
Done.